src/target/page/TargetPage.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import api, { Target } from '../../common/utils/api/api';
+import TargetPage from './TargetPage';
+
+const TEST_TARGET: Target = {
+  id: 'tprek:1000',
+  name: 'Test target',
+  address: 'Main street, Helsinki',
+  description: 'Test target description',
+};
+
+describe(`<TargetPage />`, () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('should show loading indicator', async () => {
+    jest.spyOn(api, 'getTarget').mockImplementation(() =>
+      // eslint-disable-next-line prefer-promise-reject-errors
+      Promise.resolve(TEST_TARGET)
+    );
+
+    const targetPage = mount(<TargetPage id="tprek:8100" />);
+
+    await act(async () => {
+      targetPage.update(); // First tick to trigger useEffect to load
+    });
+
+    expect(targetPage.find('p').text()).toEqual(
+      'Toimipisteen tietoja ladataan...'
+    );
+  });
+
+  test('should show error notification', async () => {
+    jest.spyOn(api, 'getTarget').mockImplementation(() =>
+      // eslint-disable-next-line prefer-promise-reject-errors
```

Comment:
Should we get rid of this eslint-disable-next-line by introducing an error object inside the `Promise.reject` like this:

`Promise.reject(new Error('Failed to load a target'))`

Reply:
Ok, I'll try